frontend/src/urls.ts

```diff
@@ -19,9 +19,6 @@ export const DEPLOY_URL = "https://share.streamlit.io/deploy"
 export const STREAMLIT_CLOUD_URL = "https://streamlit.io/cloud"
 export const ONLINE_DOCS_URL = "https://docs.streamlit.io"
 export const COMMUNITY_URL = "https://discuss.streamlit.io"
```

Comment:
Nit: Is this being used anywhere? I would assume that it's not because the linter is not catching it.

Reply:
It's used here: https://github.com/streamlit/streamlit/blob/develop/frontend/src/components/widgets/CustomComponent/ComponentInstance.tsx#L330